nui/src/main/java/org/terasology/nui/Color.java

```diff
@@ -41,19 +43,41 @@
  *
  */
 @API
-public class Color {
+public class Color implements Colorc{
 
+    @Deprecated
```

Comment:
Deprecating these sounds weird to me - isn't `Color` the class we want to use?

Reply:
See comment below, confusion about `Colorc` was resolved. Just to ensure I get this right - in case I want to have a half-transparent green somewhere I'd need to do something like this, right:

```java
Color transparentGreen = new Color(Color.green).setAlpha(0.5f);
```

Would it make sense to offer both a read-only value for the base colors and a static method that returns a new modifiable value? So for each base we would have a pair of

```java
public static final Colorc WHITE = new Color(0xFFFFFFFF);

public static Color white() {
	return new Color(WHITE);
}
```

The example above could then become

```java
Color transparentGreen = Color.green().setAlpha(0.5f);
```